src/instanceMethods/close.js

```diff
@@ -70,18 +70,23 @@ export function close (resolveValue) {
 
   handlePopupAnimation(this, popup, innerParams)
 
+  // Resolve Swal promise
+  swalPromiseResolve(finalizeResolveValue(resolveValue))
+}
+
+const finalizeResolveValue = (resolveValue) => {
   if (typeof resolveValue !== 'undefined') {
     resolveValue.isDismissed = typeof resolveValue.dismiss !== 'undefined'
-    resolveValue.isConfirmed = typeof resolveValue.dismiss === 'undefined'
+    resolveValue.isDenied = resolveValue.value === false
```

Comment:
Worrisome that this line relies on the fact that currently there is no way for a "confirmed" popup to have a `value` of `false`. 

If in the future it would be possible to get a `value` of `false` from an html input, or from `preConfirm`, or from any other source, this would need to be refactored, or else bug.

Reply:
This code also wouldn't work if we added a `preDeny` parameter that (like `preConfirm`) allowed the user to define the `value`.

I mean, there's no bug as-is (assuming no html input value can be `false`) but I think there is a more correct way to do it.
We already know the "fate" from when the popup close was initiated. That info goes into determining the `value`. We should use that info directly instead of trying to work backwards from `value`.

Reply:
Done 👍 

Your reviews are extremely helpful and will ensure the painless future of the Deny button maintanance. Thank you!